webots_ros2_ur_e_description/README.md

```diff
@@ -0,0 +1,7 @@
+# Sources
+
+These files are copied from [ros-industrial/universal_robot](https://github.com/ros-industrial/universal_robot/tree/kinetic-devel/ur_e_description).
```

Comment:
Suggestion: use a permalink. That way it's completely clear which specific version of the files you've copied.

Reply:
I'd also be specific about *which* files you've copied.

It's really only the robot `.urdf`s/`.xacro`s afaiu.